ci/build_setup.sh

```diff
@@ -80,13 +80,14 @@ if [[ -d /bazel-prebuilt-output && ! -d "${TEST_TMPDIR}/_bazel_${USER}" ]]; then
 fi
 
 # Setup Envoy consuming project.
+rm  -rf "${ENVOY_FILTER_EXAMPLE_SRCDIR}"
 if [[ ! -a "${ENVOY_FILTER_EXAMPLE_SRCDIR}" ]]
 then
-  git clone https://github.com/envoyproxy/envoy-filter-example.git "${ENVOY_FILTER_EXAMPLE_SRCDIR}"
+  git clone https://github.com/PiotrSikora/envoy-filter-example.git "${ENVOY_FILTER_EXAMPLE_SRCDIR}"
```

Comment:
Filter example is merged now; please fix.

Reply:
Done, thanks for quick turnaround!